ui/app/AppLayouts/WalletV2/components/AddAccountPopup.qml

```diff
@@ -0,0 +1,227 @@
+        if (regex.test(popup.keyOrSeedInputError)) {
+            popup.keyOrSeedInputError = qsTrId("custom-seed-phrase") + '. ' +
+                    qsTrId("custom-seed-phrase-text-1");
+        }
+        return (popup.keyOrSeedInputError === "");
+    }
+
+    onOpened: {
+        popup.reset();
+        keyOrSeedPhraseInput.forceActiveFocus(Qt.MouseFocusReason);
+    }
+
+    Item {
+        width: parent.width
+        height: childrenRect.height
+        anchors.verticalCenter: parent.verticalCenter
+        Input {
+            id: keyOrSeedPhraseInput
+            anchors.left: parent.left
+            anchors.leftMargin: 8
+            anchors.right: parent.right
+            anchors.rightMargin: 10
+            textField.anchors.rightMargin: (checkmark.width + 7)
+            customHeight: (textField.contentHeight + 34)
+            anchors.top: parent.top
+            placeholderText: qsTr("Enter private key or seed phrase")
+            label: qsTr("Private key or seed phrase")
+            validationError: popup.keyOrSeedInputError
+            onTextChanged: {
+                popup.seedPhraseInserted = keyOrSeedPhraseInput.text.includes(" ");
+                if (text === "") {
+                    popup.keyOrSeedInputError = "";
+                }
+            }
+        }
+        StatusIcon {
+            id: checkmark
+            width: 15
+            height: 11
+            anchors.right: keyOrSeedPhraseInput.right
+            anchors.rightMargin: 12
+            anchors.verticalCenter: keyOrSeedPhraseInput.verticalCenter
+            anchors.verticalCenterOffset: checkmark.height
+            visible: popup.seedPhraseInserted ? (keyOrSeedPhraseInput.text.match(/(\w+)/g).length === 12) :
+                                                Utils.isPrivateKey(keyOrSeedPhraseInput.text)
+            icon: "checkmark"
+            color: Theme.palette.primaryColor1
+        }
+
+        Separator {
+            id: separator
+            width: keyOrSeedPhraseInput.width
+            anchors.top: keyOrSeedPhraseInput.bottom
+            anchors.topMargin: popup.marginBetweenInputs
+        }
+
+        Row {
+            id: accountNameInputRow
+            anchors.left: parent.left
+            anchors.leftMargin: 8
+            anchors.right: parent.right
+            anchors.rightMargin: 10
+            anchors.top: separator.bottom
+            anchors.topMargin: popup.marginBetweenInputs
+            height: 82
+            spacing: 10
+            Item {
+                implicitWidth: 434
+                height: parent.height
+                Input {
+                    id: accountNameInput
+                    anchors.fill: parent
+                    customHeight: 56
+                    placeholderText: qsTrId("enter-an-account-name...")
+                    label: qsTrId("account-name")
+                    validationError: popup.accountNameValidationError
+                }
```

Comment:
This should be can be done using `StatusInput`. Probably need to apply a `StatusValidator` as well

Reply:
done